src/main/java/nl/tudelft/ti2206/group9/audio/SoundEffectPlayer.java

```diff
@@ -106,4 +112,19 @@ public final void setPath(final String location) {
         }
     }
 
+    @Override
+    public void setVolume(final double volumeLevel) {
+        try {
+        audioClip.setVolume(volumeLevel);
+        } catch (NullPointerException ne) { // NOPMD
+            // This try-catch block is just here for testing.
+            // The setVolume method can result in a NullPointer (according to
+            // JUnit), because JUnit can't really play audio neither can Travis.
```

Comment:
This is true, but it is not needed to set the volume of the clip when it isn't initialized. Therefore, it is better to check `if (audioClip == null)` than to catch a NPException. Catching a NPException is a bad practice, for this reason.
Also, when an `audioClip` is initialized, its volume is set immediately, so you don't have to worry about that either. :)

<review, logic